app/test/process_proposal_test.go

```diff
@@ -49,6 +50,26 @@ func TestProcessProposal(t *testing.T) {
 			testfactory.RandomBlobNamespaces(tmrand.NewRand(), 4),
 			[][]int{{100}, {1000}, {420}, {300}},
 		),
+		blobfactory.DefaultTxOpts()...,
+	)
+
+	largeMemo := strings.Repeat("a", appconsts.MaxTxBytes(appconsts.LatestVersion))
+
+	// create 2 single blobTxs that include a large memo making the transaction
+	// larger than the configured max tx bytes
+	largeBlobTxs := blobfactory.ManyMultiBlobTx(
+		t, enc, kr, testutil.ChainID, accounts[3:], infos[3:],
+		blobfactory.NestedBlobs(
+			t,
+			testfactory.RandomBlobNamespaces(tmrand.NewRand(), 4),
+			[][]int{{100}, {1000}, {420}, {300}},
+		),
+		user.SetMemo(largeMemo))
```

Comment:
[question] how does this function create 2 single blobTxs? I don't see `2` used as an argument to `ManyMultiBlobTx`

Reply:
I'll create as many blobs as the accounts you pass into the function